src/daemon/command_line_args.h

```diff
@@ -86,10 +84,6 @@ namespace daemon_args
     "daemon_command"
   , "Hidden"
   };
-  const command_line::arg_descriptor<bool> arg_os_version = {
-    "os-version"
-  , "OS for which this executable was compiled"
-  };
```

Comment:
This flag was so stupid.  It didn't actually do what it said on the tin, it actually prints the *current* system it is *running* on.